src/gwt/org/bbop/apollo/gwt/client/AnnotatorPanel.java

```diff
@@ -732,7 +751,17 @@ private void buildAnnotationRow(final AnnotationInfo rowValue, int absRowIndex,
                 SafeHtml htmlString = new SafeHtmlBuilder().appendHtmlConstant(html.getHTML()).toSafeHtml();
                 td.html(htmlString);
             } else {
-                renderCell(td, createContext(0), nameColumn, rowValue);
+                String type = rowValue.getType();
+                if (type.equals("gene") || type.equals("pseudogene")) {
+                    renderCell(td, createContext(0), nameColumn, rowValue);
+                }
+                else {
+                    // handles singleton features
+                    String featureStyle = "color: #800080;";
```

Comment:
You could try using the handler above (for the pseudogenes), but I'm not sure that they would work, though:

```
 singleSelectionModel.addSelectionChangeHandler(new SelectionChangeEvent.Handler() {
            @Override
            public void onSelectionChange(SelectionChangeEvent event) {
                selectedAnnotationInfo = singleSelectionModel.getSelectedObject();
```